lib/slack-ruby-bot/commands/help.rb

```diff
@@ -1,8 +1,42 @@
 module SlackRubyBot
   module Commands
     class Help < Base
-      def self.call(client, data, _match)
-        client.say(channel: data.channel, text: 'See https://github.com/dblock/slack-ruby-bot, please.', gif: 'help')
+      help do
+        title 'help'
+        desc 'Shows help information.'
+      end
+
+      command 'help' do |client, data, match|
+        command = match[:expression]
+
+        text = if command.present?
+                 CommandsHelper.instance.command_full_desc(command)
+               else
+                 general_text
+               end
+
+        client.say(channel: data.channel, text: text, gif: 'help')
+      end
+
+      class << self
+
+        private
```

Comment:
Really, it's possible. http://domon.cc/2013/12/25/private-class-methods-in-ruby/
I already have done it here: https://github.com/dblock/slack-ruby-bot/blob/master/lib/slack-ruby-bot/commands/base.rb#L81 😃